wallet/src/de/schildbach/wallet/util/CrashReporter.java

```diff
@@ -221,10 +221,6 @@ public static void appendApplicationInfo(
         report.append("Time of seed backup: "
                 + (lastBackupSeedTime > 0 ? String.format(Locale.US, "%tF %tT %tZ", calendar, calendar, calendar) : "none")
                 + "\n");
-        final long lastRestoreTime = configuration.getLastRestoreTime();
-        calendar.setTimeInMillis(lastRestoreTime);
-        report.append("Time of last restore: ").append(lastRestoreTime > 0 ? String.format(Locale.US, "%tF %tT %tZ",
-                calendar, calendar, calendar) : "none").append("\n");
```

Comment:
`LastRestoreTime` is never set, so the values we had in the report are bogus anyway. If we still need it, we can restore it in a datastore config class and figure out where it was set